statistics-dashboard-common/src/main/java/statistics/dashboard/common/models/filters/StatisticsRangeFilter.java

```diff
@@ -0,0 +1,40 @@
+package statistics.dashboard.common.models.filters;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+/**
+ * Class that represents a filter type range.
+ * That is, a filtering that contains range conditions
+ */
+
+@JsonSerialize
+public class StatisticsRangeFilter implements StatisticsFilter{
+
+  private String from;
+  private String to;
+
+  public StatisticsRangeFilter(){
+
+  }
```

Comment:
Perhaps, in all request objects where you have these empty constructors, make a note (javadoc) that the empty constructor is needed for deserialization?

Reply:
Will add the comments